README.md

```diff
@@ -111,26 +111,21 @@ conda install "jupyterlab>=3" "ipywidgets>=7.6"
 
 The instructions above apply to JupyterLab 3.x. **For JupyterLab 2 or earlier**, run the following commands to install the required JupyterLab extensions (note that this will require [`node`](https://nodejs.org/) to be installed):
 
-```
-# JupyterLab 2.x renderer support
-jupyter labextension install jupyterlab-plotly@5.24.1 @jupyter-widgets/jupyterlab-manager
-```
-
 Please check out our [Troubleshooting guide](https://plotly.com/python/troubleshooting/) if you run into any problems with JupyterLab.
 
-### Jupyter Notebook Support
+### Jupyter Widget Support
 
-For use in the Jupyter Notebook, install the `notebook` and `ipywidgets`
+For use as a Jupyter widget, install `jupyter` and `anywidget`
```

Comment:
The section above this talks about "JupyterLab Support" and should now mention anywidget. Wondering if this section should stay as "Jupyter Notebook Support"

Reply:
I think maybe they should be combined. The usage is almost identical for jupyter Notebook and JupyterLab